verilog/tools/kythe/kythe_schema_constants.h

```diff
@@ -31,6 +31,7 @@ constexpr absl::string_view kNodeBuiltin = "tbuiltin";
 constexpr absl::string_view kSubkindModule = "module";
 constexpr absl::string_view kCompleteDefinition = "definition";
 constexpr absl::string_view kNodeVariable = "variable";
+constexpr absl::string_view kNodeFunction = "function";
```

Comment:
Could we add

```
constexpr absl::string_view kNodeTask = "task";
```

Reply:
No Kythe doesn't have something called tasks.
We are treating tasks as functions in Kythe.

Reply:
Ok, we'll make distinguishing between functions/tasks a future issue.